src/conspiracies/prompt_relation_extraction/data_classes.py

```diff
@@ -89,13 +116,37 @@ class Config:
     subject: Span
     predicate: Span
     object: Span
-    span: Union[Span, Doc]
 
     @property
     def triplet(self) -> Tuple[Span, Span, Span]:
         """Returns the triplet as a tuple."""
         return self.subject, self.predicate, self.object
 
+    @property
+    def span(self) -> Span:
+        """Returns the span of the triplet."""
+        min_start = min([s.start for s in self.triplet])
+        max_end = max([s.end for s in self.triplet])
+
+        return self.subject.doc[min_start:max_end]
+
+    @property
+    def sentence(self) -> Span:
+        """Returns the sentence of the triplet."""
+        span = self.span
+        doc = span.doc
+
+        for sent in doc.sents:
+            if sent.start <= span.start and sent.end >= span.end:
+                return sent
+        else:
+            return doc[:]
```

Comment:
shouldn't this else clause be indented?

Reply:
Yes, it should (but apparently for else is a valid python syntax).